docker-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
as I can't comment on the file itself,
chmod +x docker-entrypoint.sh to set the proper permissions